x-pack/platform/plugins/private/discover_enhanced/ui_tests/parallel_tests/value_suggestions.spec.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { expect, tags } from '@kbn/scout';
-import { spaceTest, testData, assertionMessages } from '../fixtures';
+import { assertionMessages, spaceTest, testData } from '../fixtures';
```

Comment:
Also wondering about the changed import order. We don't have that in main, so it might cause conflicts in future backports?